ovn_k8s/common/kubernetes.py

```diff
@@ -17,22 +17,60 @@
 
 import ovs.vlog
 
+from os import path
 from ovn_k8s.common import exceptions
+from ovn_k8s.common.util import ovs_vsctl
 
+CA_CERTIFICATE = "/usr/share/openvswitch/k8s-ca.crt"
 vlog = ovs.vlog.Vlog("kubernetes")
 
 
+def _get_api_params():
```

Comment:
I was suggesting that once you retrieve data from the db you can store them in a local variable and avoid going back to the db every time we need to query the k8s API server.

I think I phrased my previous comment in a fairly crappy way ;)